### PR TITLE
ISPN-12133 Support XML Fragments in CacheContainerAdmin

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-12133

### CODE REVIEW

File: core/src/main/java/org/infinispan/configuration/parsing/FragmentParser.java [hunk 1]
```diff
@@ -0,0 +1,1911 @@
...
+@MetaInfServices(ConfigurationParser.class)
+@Namespace(root = "local-cache")
+@Namespace(root = "distributed-cache")
+@Namespace(root = "replicated-cache")
+@Namespace(root = "scattered-cache")
+@Namespace(uri = NAMESPACE + "*", root = "local-cache")
+@Namespace(uri = NAMESPACE + "*", root = "distributed-cache")
+@Namespace(uri = NAMESPACE + "*", root = "replicated-cache")
+@Namespace(uri = NAMESPACE + "*", root = "scattered-cache")
+public class FragmentParser implements ConfigurationParser {
```
Comment (ryanemerson) [inline core/src/main/java/org/infinispan/configuration/parsing/FragmentParser.java:72]: How about `CacheConfigParser` or `CacheParser`?

Comment (tristantarrant) [reply core/src/main/java/org/infinispan/configuration/parsing/FragmentParser.java]: `CacheParser` it is. Updated.

**Comment by tristantarrant** [comment]
Don't merge before https://github.com/infinispan/infinispan/pull/8815
It's easier to change here than there

**Comment by tristantarrant** [comment]
Rebased

**Review by oraNod — COMMENTED** [review]
@tristantarrant  This change should be reflected in the simple tutorials. Should we include that in this PR?

**Comment by tristantarrant** [comment]
@ryanemerson I've applied Don's suggestions

**Review by karesti — APPROVED** [review]

**Pull request merged**